cli/src/pcluster/aws/fsx.py

```diff
@@ -43,6 +45,44 @@ def get_file_systems_info(self, fsx_fs_ids):
                 result.append(file_system_info)
         return result
 
+    @AWSExceptionHandler.handle_client_exception
+    def describe_storage_virtual_machines(self, storage_virtual_machine_ids):
+        """Describe storage virtual machines."""
+        result = []
+        missed_storage_virtual_machine_ids = []
+        for storage_virtual_machine_id in storage_virtual_machine_ids:
+            cached_data = self.svm_cache.get(storage_virtual_machine_id)
+            if cached_data:
+                result.append(cached_data)
```

Comment:
Got it. So, within a single pcluster command how many times are we expecting to execute such API calls to justify a caching strategy?

Reply:
Ok, so please correct me if I'm wrong.

We are caching SVM data because we can mount different volumes having the same SVM, so with caching we are limiting the number of times we retrieve SVM data.

Correct?